packages/dependency-extraction-webpack-plugin/lib/index.js

```diff
@@ -2,8 +2,10 @@
  * External dependencies
  */
 const { createHash } = require( 'crypto' );
+// @ts-ignore
 const json2php = require( 'json2php' );
 const path = require( 'path' );
+// @ts-ignore
 const { ExternalsPlugin } = require( 'webpack' );
```

Comment:
Adding in to webpack types in https://github.com/DefinitelyTyped/DefinitelyTyped/pull/45179